resources/assets/components/ReviewBlock/test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+
+import ReviewBlock from './index';
+
+test('it renders accepted, rejected, and delete buttons', () => {
+    const post = {
+      status: 'pending',
+    };
+
+  const component = shallow(
+    <ReviewBlock post={post} />
+  );
+
+  expect(toJson(component)).toMatchSnapshot();
+});
+
+test('it renders the review block with tags', () => {
+    const post = {
+      status: 'accepted',
+    };
+
+  const component = shallow(
+    <ReviewBlock post={post} />
+  );
+
+  expect(toJson(component)).toMatchSnapshot();
+});
+
+test('it renders an active Accept button when clicked', () => {
+  const post = {
+    status: 'pending',
+  };
+
+  const component = shallow(
+    <ReviewBlock post={post} />
+  );
+
+    // Click the "Accept" button.
+    component.find('.accepted').first().simulate('click');
+
+    // It should now show the active "Accept" button.
+    expect(component.find('.accepted').first().hasClass('button -accepted accepted'));
```

Comment:
Oh hmm, this whole `-accepted accepted` and `-rejected rejected` thing is pretty weird. Maybe we can switch those `accepted` and `rejected` classes to `is-selected` or something more clear?

I also noticed most of the [styles](https://github.com/DoSomething/rogue/blob/31cf6bf3a8d882527dde1135d757a464756ffc0f/resources/assets/components/StatusButton/status-button.scss) for those classes are the same, can we perhaps join them into a `outlined-button` class (and only have specific overrides for the hover state)?